pyht/async_client.py

```diff
@@ -141,15 +141,15 @@ async def warmup(self):
         try:
             async with aiohttp.ClientSession() as session:
                 assert self._inference_coordinates is not None, "No connection"
-                async with session.options(self._inference_coordinates.address,
+                async with session.options(self._inference_coordinates["Play3.0-mini"]["http_streaming_url"],
```

Comment:
Are you guys running this on Python 3? If so, i'd **highly recommend** using the following pattern for a better key guarantees. 
```suggestion
                async with session.options(self._inference_coordinates.get("Play3.0-mini", {}).get("http_streaming_url", "")
```

Reply:
See other comments about what you want to see during an error event. There's an opportunity to be hyper targeted versus generally ambiguous when error'ing.